net-mail/mu/mu-1.4.1.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools elisp-common
+
+DESCRIPTION="Set of tools to deal with Maildirs, in particular, searching and indexing"
+HOMEPAGE="http://www.djcbsoftware.nl/code/mu/ https://github.com/djcb/mu"
+SRC_URI="https://github.com/djcb/mu/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="emacs guile mug"
+
+DEPEND="
+	dev-libs/glib:2
+	dev-libs/gmime:3.0
+	>=dev-libs/xapian-1.4
+	emacs? ( >=app-editors/emacs-24.4:* )
+	guile? ( >=dev-scheme/guile-2.2 )
+	mug? (
+		net-libs/webkit-gtk:4
+		x11-libs/gtk+:3
+	)"
+RDEPEND="${DEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+SITEFILE="70mu-gentoo-autoload.el"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	local myeconfargs=(
+		$(use_enable emacs mu4e)
+		$(use_enable mug gtk)
+		$(use_enable mug webkit)
```

Comment:
Should the 'mug' USE be renamed as 'gtk'? I think it'd make more sense, then it's controllable profile-wide easier.

Reply:
I'm not sure, the documentation suggests that mug is just a toy and shouldn't be packaged. Not the most useful of tools